core/src/wallet/cosmos/explorers/RpcsParsers.hpp

```diff
@@ -461,8 +478,36 @@ namespace ledger {
                         index++;
                     }
                 }
+
                 assert((vNode.HasMember(kFee)));
                 parseFee(vNode[kFee].GetObject(), transaction.fee);
+
+                // TODO(remibarjon): create a function to post treat the transaction after it is parsed
+                // (in the function calling this function).
+                const auto index = transaction.messages.size();
+                auto msgFeesContent = cosmos::MsgFees();
+                auto pubKey = std::string();
+                parseSignerPubKey(vNode, pubKey);
+                const auto decoded = cereal::base64::decode(pubKey);
+                const auto vPubKey = std::vector<uint8_t>(std::begin(decoded), std::end(decoded));
+                msgFeesContent.payerAddress = CosmosLikeKeychain(vPubKey,
+                        DerivationPath(""), currencies::ATOM).getAddress()->toBech32();
+                const auto fees =
```

Comment:
Can you add a comment to say you're filtering out the non "uatom" fees ? 2 reasons :

- the `assert` doesn't exist in non-debug builds
- It's easier to know we can grep for a comment if we need to understand if something weird happened to the fees.

To be clear, I don't expect that cosmoshub starts taking fees in any other currency than uatom. It's just some future-proofing